etc/helm/pachyderm/values.yaml

```diff
@@ -690,6 +690,9 @@ pachw:
   # when enabled, pachw instances can still run in their own dedicated kubernetes deployment if maxReplicas is greater than 0.
   # For more control of where pachw instances run, 'inSidecars' should be disabled.
   inSidecars: false
+  # When set to true, preprocessing tasks are only processed by pachw instances.
+  # When set to false, pipeline workers will also participate in processing preprocessing tasks.
+  onlyPreprocessing: true
```

Comment:
The name of this flag may lead to confusion (as well as `PACHW_ONLY_PREPROCESSING`). Without the comment, I'd interpret this as pachw only processes preprocessing tasks and no other kinds of tasks.
That said, I'm not sure what a better name for this is. Maybe `handleAllPreprocessingTasks`?

Reply:
Yeah, I was actually going to bring this up in standup. I don't really like the name either. I'll wait until then when we can discuss it before merging.